### PR TITLE
text1 file

### PR DESCRIPTION
**Pull request by younus1804**

### CODE REVIEW

**Comment by RichMans96** [comment]
this is just a test blah blah blah

File: newfile1.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+Text example
```
Comment (shalamike) [inline newfile1.txt:1]: another change by mike

**Review by RichMans96 — COMMENTED** [review]
Test review

File: newfile1.txt (hunk 1 above)
Comment (shalamike) [reply newfile1.txt:1]: and another comment

**Pull request merged**